Flaky `e2e` create migration from anywhere test failing due to cross-contamination with other tests

Another flaky test [related to the create migration from anywhere happened](https://github.com/serverpod/serverpod/actions/runs/20829723988/job/59840486897?pr=4553) on PR #4553:

```console
+9: test/generate_interactive_flag_test.dart: Given multiple server directories when generate is called then with --interactive it shows selection prompt
Output when multiple servers found:
Multiple Serverpod server projects found. Select one:
(●) /tmp/dart_test_LEUZMC/multi_server_project/server2
(○) /tmp/dart_test_LEUZMC/multi_server_project/server1

Press [Enter] to confirm.
ERROR: StdinException: Error getting terminal echo mode, OS Error: Inappropriate
ioctl for device, errno = 25
+11 -1: test/create_migration_from_anywhere_test.dart: Given a serverpod project without a root pubspec when create-repair-migration is called from client directory then it should not fail due to package detection. [E]
  TimeoutException after 0:01:30.000000: Test timed out after 1 minutes, 30 seconds.
  dart:isolate  _RawReceivePort._handleMessage
  
+11 -1: Some tests failed.

Consider enabling the flag chain-stack-traces to receive more detailed exceptions.
For example, 'dart test --chain-stack-traces'.
Error: Process completed with exit code 1.
```

The timeout failure happened again on this test on the job [CLI e2e tests (3.32.0, ubuntu-latest, 1, tests/serverpod_cli_e2e_test)](https://github.com/serverpod/serverpod/actions/runs/21518539409/job/62009500609?pr=4664#logs). I had to run the CI 4 times, with 3 failing with the same message:

```console
✓ Generating code (68ms)
Incremental code generation complete.
+10 -1: test/create_migration_from_anywhere_test.dart: Given a serverpod project with a root pubspec when create-repair-migration is called from project root then it should not fail due to root pubspec. [E]
  TimeoutException after 0:01:30.000000: Test timed out after 1 minutes, 30 seconds.
  dart:isolate  _RawReceivePort._handleMessage
  
+10 -2: test/create_migration_from_anywhere_test.dart: Given a serverpod project without a root pubspec when create-migration is called from client directory then it should succeed. [E]
  TimeoutException after 0:01:30.000000: Test timed out after 1 minutes, 30 seconds.
  dart:isolate  _RawReceivePort._handleMessage
  
+10 -2: Some tests failed.

Consider enabling the flag chain-stack-traces to receive more detailed exceptions.
For example, 'dart test --chain-stack-traces'.
Error: Process completed with exit code 1.
```

--- a/tests/serverpod_cli_e2e_test/test/create_migration_from_anywhere_test.dart
+++ b/tests/serverpod_cli_e2e_test/test/create_migration_from_anywhere_test.dart
@@ -1,3 +1,4 @@
+@Timeout(Duration(minutes: 5))
 import 'package:path/path.dart' as path;
 import 'package:serverpod_cli_e2e_test/src/run_serverpod.dart';
 import 'package:test/test.dart';

--- a/tests/serverpod_cli_e2e_test/test/generate_from_anywhere_test.dart
+++ b/tests/serverpod_cli_e2e_test/test/generate_from_anywhere_test.dart
@@ -1,3 +1,4 @@
+@Timeout(Duration(minutes: 5))
 import 'dart:io';
 
 import 'package:path/path.dart' as path;

